### PR TITLE
Merge upstream E3SM 12 Nov 2019

### PR DESCRIPTION
**Pull request by ambrad**
Bring in recent HOMME theta-l NH changes.

### CODE REVIEW

**Comment by ambrad** [comment]
@jgfouca go ahead and merge this PR if you approve.

File: components/clm/tools/clm4_5/OLMT/runcase.py [hunk 1]
```diff
@@ -0,0 +1,1591 @@
+#!/usr/bin/env python
```
Comment (brhillman) [inline components/clm/tools/clm4_5/OLMT/runcase.py]: There's a lot of files in this directory that look like they just have additions (new files?). Is it possible that this was a submodule that accidentally had the actual source copied over and committed to the repo?

Comment (ambrad) [reply components/clm/tools/clm4_5/OLMT/runcase.py]: Are you referring to clm4_5 and deeper? These files exist in the E3SM repo as regular files; see, e.g., https://github.com/E3SM-Project/E3SM/tree/master/components/clm/tools/clm4_5. Plus these are all mod'ed files, not new ones, unless I'm missing something.

Comment (brhillman) [reply components/clm/tools/clm4_5/OLMT/runcase.py]: Sorry was referring to the OLMT directory. Some of these looked like new files based on the git diffs. Looks like these exist in the E3SM repo though, so I retract my comment.

**Pull request merged**